src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java

```diff
@@ -0,0 +1,12 @@
+package com.github.seratch.jslack.api.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ErrorMetadata {
```

Comment:
How about renaming this to `com.github.seratch.jslack.api.model.dialog.DialogResponseMetadata`?

Reply:
Please change its package as well.

Reply:
Please change its package as well.